### DISCUSSION TITLE
GitHub App authentication + incorrect username = silent fail when trying to update content of existing PR

### DISCUSSION THREAD

**Discussion by walczakp** [renovatebot/renovate#12168] [Suggest an Idea]
Hello!
We stumbled upon a very interesting issue when running self-hosted Renovate 27.31.10 on GitHub repos using GH App for authentication. In following scenario: 
* there is already an unmerged PR with update of a dependency created by renovate
* this dependency gets updated upstream once again (e.g. on maven central)

Renovate will not update PR (title, body, ...), if the username specified in config does not match the format `our-cool-app-name[bot]`.
After digging deeper through trace logs, I noticed Renovate returning 0 found PRs when `/repos/{owner}/{repo}/pulls` is called, which in turn causes a new PR creation request to which GitHub responds with 422. This is unfortunately logged only on debug level. I believe this is due to github returning app name + `[bot]` suffix when a request to `/repos/{owner}/{repo}/pulls` is made (field user->login).

I'm aware (correct me please if I'm wrong) that this is an expected behavior of Renovate (to find only PRs with username matching the configuration), but I think it may be a good idea to document this GitHub's quirk somewhere in Renovate docs. It would spare the next team digging through this some time. What's your take on that?

**Comment by viceice** [comment]
https://docs.renovatebot.com/getting-started/running/#running-as-a-github-app ?

**Reply by walczakp** [reply]
Darn! I searched documentation for something completely different at the start of investigation and I missed that section completely... Thanks for pointing this out.
